canadian-journal-economics-default.csl

```diff
@@ -0,0 +1,611 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="never" version="1.0" class="in-text" page-range-format="chicago">
+  <info>
+    <title>Canadian Journal of Economics: InText Default (Author-Date)</title>
+    <id>http://www.zotero.org/styles/canadian-journal-economics-default</id>
+    <link rel="self" href="http://www.zotero.org/styles/canadian-journal-economics-default"/>
+    <link rel="documentation" href="http://www.chicagomanualofstyle.org/tools_citationguide.html"/>
+    <author>
+      <name>Julian Onions</name>
+      <email>julian.onions@gmail.com</email>
+    </author>
+    <contributor>
+      <name>Sebastian Karcher</name>
+    </contributor>
+    <contributor>
+      <name>Richard Karnesky</name>
+      <email>karnesky+zotero@gmail.com</email>
+      <uri>http://arc.nucapt.northwestern.edu/Richard_Karnesky</uri>
+    </contributor>
+    <contributor>
+      <name>Naomi Scott</name>
+      <uri>http://www.mendeley.com/profiles/naomi-scott/</uri>
+    </contributor>
+    <contributor>
+      <name>Andrew Dunning</name>
+      <email>andrew.dunning@utoronto.ca</email>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="generic-base"/>
```

Comment:
Please change the field to something more appropriate. The options (http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories) are rather limited, but "social_science" is better than "generic-base")